terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java

```diff
@@ -32,14 +32,14 @@
 @Component
 public class ReservationReportPdfStamperView extends AbstractPdfStamperView {
 
-    private static final SimpleDateFormat SDF = new SimpleDateFormat("yyyy/MM/dd");
-
     private static final String DOWNLOAD_PDF_EXTENSION = ".pdf";
 
     private static final int REFERENCE_NAME_MAXIMUM_VALUE_WITH_NORMAL_FONTSIZE = 30;
 
     private static final float REFERENCE_NAME_VARIABLE_FONTSIZE = 8.0F;
 
+    private final SimpleDateFormat sdf = new SimpleDateFormat("yyyy/MM/dd");
```

Comment:
- [ ] I essentially recommend using JSR-310 to format with DateTimeFormatter. (Not a comment that mandatory to fix)

Reply:
We have created another Issue https://github.com/terasolunaorg/terasoluna-tourreservation/issues/676